libraries/AP_HAL_ChibiOS/hwdef/common/common_extf_h750.ld

```diff
@@ -249,13 +266,13 @@ SECTIONS
     .ARM.extab :
     {
         *(.ARM.extab* .gnu.linkonce.armextab.*)
-    } > default_flash
```

Comment:
a comment about this would be good

Reply:
Will do. It needs to be in RAM because libgcc.a was moved there